xivo/mallow/fields.py

```diff
@@ -1,23 +1,33 @@
 # Copyright 2017-2021 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from marshmallow.fields import (
-    Boolean as _Boolean,
-    Constant as _Constant,
-    Date as _Date,
-    DateTime as _DateTime,
-    Dict as _Dict,
-    Email as _Email,
-    Field as _Field,
-    Float as _Float,
-    Integer as _Integer,
-    List as _List,
-    Nested as _Nested,
-    String as _String,
-    TimeDelta as _TimeDelta,
-    URL as _URL,
-    UUID as _UUID,
-)
+try:
+    from marshmallow.fields import (
+        Boolean as _Boolean,
+        Constant as _Constant,
+        Date as _Date,
+        DateTime as _DateTime,
+        Dict as _Dict,
+        Email as _Email,
+        Field as _Field,
+        Float as _Float,
+        Integer as _Integer,
+        List as _List,
+        Nested as _Nested,
+        String as _String,
+        TimeDelta as _TimeDelta,
+        URL as _URL,
+        UUID as _UUID,
+    )
+except Exception:
+    import sys
+
+    python_major_version = sys.version_info.major
+    if python_major_version == 2:
+        raise ImportError(
+            "Marshamallow library is incompatible with Python version %s" % sys.version
+        )
```

Comment:
What if import doesn't trigger `Exception`? Can we use this kind of condition instead? (not tested)
```suggestion
import sys

python_major_version = sys.version_info.major
if python_major_version == 2:
    raise ImportError(
        "Marshamallow library is incompatible with Python version %s" % sys.version
    )

from marshmallow.fields import (
    Boolean as _Boolean,
    Constant as _Constant,
    Date as _Date,
    DateTime as _DateTime,
    Dict as _Dict,
    Email as _Email,
    Field as _Field,
    Float as _Float,
    Integer as _Integer,
    List as _List,
    Nested as _Nested,
    String as _String,
    TimeDelta as _TimeDelta,
    URL as _URL,
    UUID as _UUID,
)
```